docs/api/users-create-user.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+---
+
+# Create user
+
+Creates a [`user`](user) who is registering with the to-do service.
+The request body contains the new user details.
+You must specify the required properties for the user. 
+
+## URL
+
+```shell
+
+{POST}{server_url}/users/
+```
+
+## Request headers
+
+This request does not use any authorization. The endpoint is available to all users and applications.
+
+| Header name | Description | Required | Values |
+| -------------- | ------ | ------------ |------------ |
+| Content-Type | The format of the data to be posted. | Optional | application/json. Default value.  |
+| Accept | The format of the data to be returned. | Optional | application/json. Default value.|
```

Comment:
```suggestion
| Accept | The format of the data to be returned. | Optional | application/json. Default value. |
```
It looks like I missed this in the first review, but there should be a space between the text and the table formatting character. I think it's mostly for readability, but it might also confuse some markdown processors.